postBuild

```diff
@@ -0,0 +1,4 @@
+pip install matplotlib pillow
```

Comment:
Can we add a comment to this file to explain how it's used. It's not clear that this is an impl detail for how binder builds the images it uses for running notebooks. Since this is in the repo root (and not a hidden file) it'll be good to explain how it's used (and/or just link to the binder docs on the subject).

Reply:
Added in 630228a